src/analysis/AnalysisTest.java

```diff
@@ -1143,5 +1143,86 @@ public void testContributionBreakdownByDate() {
 
         assertEquals(expected, actual);
     }
+
+    /**
+     * Tests the toString method with no dates.
+     */
+    @Test
+    public void testToString() {
+        String username = "collab1";
+        String expected = String.format("Collaborator %s made 3 commits, 3 issues, and 6 comments.\n", username);
+        Repository repo = new Repository();
+        Collaborator collab1 = new Collaborator();
```

Comment:
Did not set the collaborator's name & username, so everything that counts by collaborator can't tell based on username that it is this collaborator.

Reply:
Want to create a new Collaborator with the other constructor like:
Collaborator collab1 = new Collaborator(String firstName, 
            String lastName, String userName, String id)
Except correct of course